db/seeds.rb

```diff
@@ -0,0 +1,81 @@
+# This file should ensure the existence of records required to run the application in every environment (production,
+# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Example:
+#
+#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
+#     MovieGenre.find_or_create_by!(name: genre_name)
+#   end
+
+# Create Users
+
+user1 = User.create(name: 'John Doe')
+
+user2 = User.create(name: 'Jane Doe')
+
+# Create Inventories
+
+inventory1 = Inventory.create(name: 'Fridge', user: user1)
+
+inventory2 = Inventory.create(name: 'Pantry', user: user1)
+
+Inventory.create(name: 'Fridge', user: user2)
+
+Inventory.create(name: 'Pantry', user: user2)
+
+# Create Foods
+
+food1 = Food.create(name: 'Eggs', measurement_unit: 'dozen', price_dollars: 2)
+
+food2 = Food.create(name: 'Milk', measurement_unit: 'gallon', price_dollars: 3)
+
+food3 = Food.create(name: 'Bread', measurement_unit: 'loaf', price_dollars: 1)
+
+# Create InventoryFoods
+
+InventoryFood.create(inventory: inventory1, food: food1, quantity: 1)
+
+InventoryFood.create(inventory: inventory1, food: food2, quantity: 1)
```

Comment:
## [OPTIONAL]

- [ ] I think it would be nice to fix this error in your seed file 

![image](https://github.com/katarighe/recipe-app/assets/65068771/62139b78-b1c1-458b-bee0-796b70e61343)